packages/cookbook/src/accounts/create-account.js

```diff
@@ -0,0 +1,62 @@
+const { connect, KeyPair, keyStores, utils } = require('near-api-js');
```

Comment:
This was originally two separate scripts with only the differences being the network specified. I consolidated them into this new file, which takes the network ID (`testnet` or `mainnet`) as a parameter but otherwise works the same.

However I was not able to write a successful integration for this against `localnet` as it appears to rely on creating sub-accounts (i.e. `Account.createAccount(...)` does work in `localnet`) rather than calling the `new_account` method on the root account.